Make common value for enabling app-mgmt and conn-svc (#2879)
kyma-project/kyma: resources/application-connector/README.md

## Details

The Application Connector Helm chart contains all the global components:
- Applicaiton broker
- Application operator
- Application registry
- Connection token handler
- Connector service

## Connector Service

> The Connector Service is responsible for generating and sending back client certificates based on Certificate Signing Request (CSR). It supports two modes: standalone and central. In standalone mode, the Connector Service works on and for the cluster in which it is deployed. In the central mode, Connector Service supports multiple clusters. By default, the installation of the Connector Service installation is set to standalone.

Reviewer: ```suggestion
The Connector Service is responsible for generating and sending back client certificates based on Certificate Signing Request (CSR). It supports two modes: standalone and central. In standalone mode, the Connector Service works on and for the cluster in which it is deployed. In the central mode, Connector Service supports multiple clusters. By default, the Connector Service installs as a standalone component.
```

Reviewer 2: This one should also be applied

> ### Install Connector Service as central

Reviewer: ```suggestion
### Install Connector Service as a standalone component
```

Author: But this applies to the Connector Service working as the central, not standalone.

> To install Connector Service configured in the central mode you must override values as presented in the following example:

Reviewer: ```suggestion
To install the Connector Service as a standalone component, you must override several values of the Kyma Installer. Run: 
```

Reviewer 2: I would suggest slight merge of two versions:
```
To install the Connector Service configured in the central mode, you must override several values of the Kyma Installer. Run: 
```

```
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: ConfigMap
metadata:
  name: connector-service-central-overrides
  namespace: kyma-installer
  labels:
    installer: overrides
    kyma-project.io/installation: ""
data:
  connector-service.deployment.args.central: "true"
  connector-service.tests.central: "true"
EOF
```

> ### Install without Connector Service

Reviewer: ```suggestion
### Install Application Connector without the Connector Service
```

Reviewer 2: Please apply those suggestions

> You can install Application Connector without Connector Service. For example in preparing an environment for working with central Connector Service. Installation without Connector Service will also disable the possibility to create Application from the UI. To perform the installation without Connector Service you must override values as presented in the following example:

Reviewer: ```suggestion
You can install the Application Connector (AC) without the Connector Service. Use this approach when preparing an environment for working with a Connector Service deployed as a standalone component. 
To install the AC without the Connector Service, you must override several values of the Kyma Installer. Run:
>**NOTE:** Installing AC without the Connector Service disables creating new Applications through the Console UI.
```

Reviewer 2: Same here :)
